src/ai/ai_help_structs.h

```diff
@@ -416,7 +416,8 @@ struct BuildableField {
 	ExtendedBool is_portspace;
 	bool port_nearby;  // to increase priority if a port is nearby,
 	// especially for new colonies
-	ExtendedBool portspace_nearby;  // special fields intended for ports
+	ExtendedBool portspace_nearby;    // special fields intended for ports
+	ExtendedBool shipyard_preferred;  // special fields intended for shipyards
```

Comment:
the new bool is set if a field is near a port field to ensure to the highest extend that we will have ship and port at the same sea